perf_test/sparse/spmv/ArmPL_SPMV.hpp

```diff
@@ -48,31 +48,32 @@
 #ifdef KOKKOSKERNELS_ENABLE_TPL_ARMPL
 #include <armpl.h>
 
-template <typename Scalar>
-void armpl_matvec_wrapper(armpl_spmat_t A, Scalar* x, Scalar* y) {
-  throw std::runtime_error(
-      "Can't use ArmPL mat-vec for scalar types other than double and float.");
-}
+// template <typename Scalar>
```

Comment:
This needs a clean-up

Reply:
@lucbv I couldn't test these changes without fixing SPMV. Sorry for the drive-by changes.